### PR TITLE
Adjustable consensus traffic controls

### PR DESCRIPTION
**Pull request by marc-casperlabs**
This PR adds configuration settings for the weight applied when throttling. The default configuration values are chosen to mirror the current production behavior.

### CODE REVIEW

File: node/CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ All notable changes to this project will be documented in this file.  The format
 * Added new event to the main SSE server stream accessed via `<IP:Port>/events/main` which emits hashes of expired deploys.
 * Added `contract_runtime_execute_block` histogram tracking execution time of a whole block.
 * Long running events now log their event type.
+* Individual weights for traffic throttling can now be set through the configuration.
```
Comment (Fraser999) [inline node/CHANGELOG.md]: We should be explicit about the new config field name.

**Review by Fraser999 — COMMENTED** [review]
We need to update two of the three config.toml files in utils/nctl/sh/scenarios/configs too: bond_its.config.toml and itst13.config.toml, but not gov96.config.toml.

File: node/CHANGELOG.md (hunk 1 above)
Comment (marc-casperlabs) [reply node/CHANGELOG.md]: Fair enough. Let's hope no one shoots themselves in the foot with this :)

**Review by Fraser999 — APPROVED** [review]

**Pull request merged**